api/src/main/resources/context.json

```diff
@@ -5,7 +5,15 @@
       "@vocab": "http://wellcomecollection.org/ontologies/works/",
       "id": "@id",
       "type": "@type",
-      "hasCreator": {
+      "creators": {
+        "@type": "wa:agent",
+        "@container": "@list"
+      },
+      "createdDate": {
+        "@type": "wa:period"
```

Comment:
This would be `"@type": "wa:Period"`, but I don't think we need it, as our style is to include the type in the response.